package.json

```diff
@@ -107,6 +107,7 @@
     "@vscode/test-electron": "^2.1.2",
     "axios": "^0.26.1",
     "glob": "^7.2.0",
-    "mocha": "^9.2.2"
+    "mocha": "^9.2.2",
+    "openai": "^2.0.5"
```

Comment:
why do we need this?

Reply:
Open ai for obvious reasons. I don't know why mocha has a line diff at all, lol

Reply:
As discussed via Zoom, it was my bad. It's removed on the latest branch version anyway